__tests__/engines.test.js

```diff
@@ -6,7 +6,7 @@ const pkg = require('../package.json');
 describe('engines.node', () => {
 	it("is the same as stylelint's one", () => {
 		const stylelintVersion = pkg.peerDependencies.stylelint;
-		const [nodeVersion] = JSON.parse(
```

Comment:
This *didn't* change the result for me on macOS, but seems to fix it on the runners. Strange?

Reply:
The test failed locally for me. This change fixed it.

Let's merge for the release. We can can revisit this change later if needed.

Reply:
As an aside, this what copilot had to say:

"The array destructuring on line 11 of the code is actually not causing the test to fail. In fact, it is necessary for the test to work correctly. 

The array destructuring syntax `[nodeVersion]` is used to extract the first element of the array returned by `JSON.parse()`. The `JSON.parse()` method is used to parse the output of the `npm view` command, which returns a JSON string containing an array of node versions that are compatible with the specified version of `stylelint`. 

Without the array destructuring, the `nodeVersion` variable would contain the entire array of compatible node versions, which would cause the test to fail because it would be comparing an array to a string. 

Therefore, the array destructuring is actually necessary for the test to work correctly."

So, something strange is going on.

Reply:
Hm, but that's not true right? In the JSON structure above, the `engines.node` key returns a string, not an array; destructuring it will only give the first character (`^`) of the string. At least on CI, I believe this is the correct behaviour.

Locally, I reinstalled `npm` and cleared the local package cache, and can now replicate the error *before* I made the change.

Reply:
> Hm, but that's not true right?

Yeap, it's likely hallucinating.

I wonder why the tests on https://github.com/stylelint/stylelint-config-recommended/pull/195 passed?

Reply:
Sorry for causing your confusion. 🙇🏼 

I've just opened PR #197 to fix the test.